README.md

```diff
@@ -19,26 +19,35 @@ limitations under the License.
 
 # Setting up the environment 
 
-```
-conda create --name iwslt-autodub --file requirements.txt
+```bash
+sudo apt install git-lfs awscli
+git lfs install
+git clone https://github.com/amazon-science/iwslt-autodub-task.git
+cd iwslt-autodub-task
+
+# Create a conda environment
+conda env create --file environment.yml
 conda activate iwslt-autodub
 
 cd <TODO_repo_dir>
+# TODO: We already pip install subword-nmt and should not need this
 git clone https://github.com/rsennrich/subword-nmt.git subword-nmt
 git clone https://github.com/moses-smt/mosesdecoder.git mosesdecoder
 
+# Download CoVoST2 TSV files
 aws s3 cp <TODO_dir1>/covost-2/covost_v2.en_de.train.tsv ./covost_tsv
```

Comment:
lets take this out and ask users to download covost data in a directory and use the TSV files extracted in that directory.

Reply:
Changed instructions to download and extract TSV files as in https://github.com/facebookresearch/covost#covost-2